recipes/rb-net-ssh/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "net-ssh" %}
+{% set version = "5.2.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 80a8a71d39e9f917ed265f66c0c5985f1849fc5e7156ac48f14dd1d4af683c80
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+
+test:
+  commands:
+    - gem --version
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: {{ name }}-{{ version }}/LICENSE.txt
+  summary: A pure-Ruby implementation of the SSH2 client protocol.
+  doc_url: https://www.rubydoc.info/gems/{{ name }}
+  dev_url: https://github.com/net-ssh/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - benhuff
```

Comment:
Can we add here `- conda-forge/ruby`? Sodre is part of this team so he could be removed.